### PR TITLE
tlg0062.tlg018_tlg062.tlg024

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updating basic cts_.xml and other metadata.

### CODE REVIEW

File: data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml [hunk 1]
```diff
@@ -4,30 +4,31 @@
...
-<title xml:lang="eng">The Cock</title>
+<title>The Cock</title>
  <author>Lucian of Samosata</author>
 <editor role="translator">Emily James Smith</editor>
  <funder>National Endowment for the Humanities</funder>
  <principal>Gregory Crane</principal>
  <respStmt>
-  <persName>Gregory Crane</persName>
-<resp>Digital conversion and editing</resp>
+  <resp>Prepared under the supervision of</resp>
```
Comment (lcerrato) [inline data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml:13]: I see this added here but not in the rest of the batch?

File: data/tlg0062/tlg022/tlg0062.tlg022.perseus-eng2.xml [hunk 2]
```diff
@@ -32,17 +32,23 @@
...
 </editor>
 <author ref="urn:cts:greekLit:tlg0062">Lucian of Samosata</author>
 <imprint>
-<publisher>Trustees of Tufts University</publisher>
-<pubPlace>Cambridge, MA</pubPlace>
+ <pubPlace>London</pubPlace>
+ <publisher>William Heinemann Ltd.</publisher>
+ <pubPlace>Cambridge, MA</pubPlace>
+ <publisher>Harvard University Press</publisher>
+ <date>1915</date>
```
Comment (lcerrato) [inline data/tlg0062/tlg022/tlg0062.tlg022.perseus-eng2.xml:44]: two dates?

**Review by lcerrato — COMMENTED** [review]
question on header changes and one duplicate

File: data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml:13]: Yes, I realized I was getting too bogged down in the Emily James Smith files and decided to do a separate pass on those to fix both cts and bigger edits (per your other issue!)

File: data/tlg0062/tlg022/tlg0062.tlg022.perseus-eng2.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0062/tlg022/tlg0062.tlg022.perseus-eng2.xml:44]: I may have just copied it in twice by accident. Thanks!

File: data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml (hunk 1 above)
Comment (lcerrato) [reply data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml:13]: ok!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**